dataset/main.py

```diff
@@ -15,12 +15,41 @@ class Flickr8kDataset(Dataset):
     Dataset for Flickr8k data treatment
     """
 
-    def __init__(self, dataset_folder=os.path.join('data'),transform=None,reduce=False,vocab_max_size=5000):
+    def __init__(self, dataset_folder=os.path.join('data'),transform=None,reduce=False,vocab_max_size=5000,split='train'):
         super(Flickr8kDataset, self).__init__()
         self.transform = transform
         self.images_folder = os.path.join(dataset_folder,'Images')
+        
         self.dataframe = pd.read_csv(open(os.path.join(dataset_folder,'captions.txt'),'r'))
 
+        # Open file with the images for train, test or eval
+        if split=='train':
+            imgs_file = open(os.path.join('data','Flickr_8k.trainImages.txt'), "r")
+        elif split=='test':
+            imgs_file = open(os.path.join('data','Flickr_8k.testImages.txt'), "r")
+        if split=='eval':
```

Comment:
`if split == 'eval'  ` change it for `else:`

Reply:
Do we really need 3 splits? I think we should be ok with 2